docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+
+  database:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_PASSWORD=development
+
+  app:
+    image: github-inbox
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - 3000:3000
```

Comment:
It would be nice to see something like `- ${GITHUB_INBOX_PORT:-3000}:3000` here, but that requires using version 2.1 of the docker-compose.yml format, which in turn requires docker-compose >= 1.9.  I'm not sure if enough people are on 1.9 yet.